docs/source/nbconvert_library.ipynb

```diff
@@ -46,8 +46,8 @@
    "source": [
     "from urllib.request import urlopen\n",
     "\n",
-    "url = 'https://jakevdp.github.io/downloads/notebooks/XKCD_plots.ipynb'\n",
-    "response = urlopen(url).read().decode()\n",
+    "url = ' https://jakevdp.github.io/downloads/notebooks/XKCD_plots.ipynb '\n",
+    "response = urlopen(url.strip()).read().decode()\n",
```

Comment:
This change shouldn't have to be made. It's only here because the link check things that it needs to check this code, and gets the link wrong.

Reply:
This is now fixed by using nbsphinx properly in 3832b3c